experimental/cc_slurm_nhc/cc_slurm_nhc/specs/default/cluster-init/files/configure_nhc.sh

```diff
@@ -87,6 +88,13 @@ function slurm_config() {
       echo "HealthCheckProgram=${NHC_EXE}" >> $SLURM_CONF
       echo "HealthCheckInterval=${SLURM_HEALTH_CHECK_INTERVAL}" >> $SLURM_CONF
       echo "HealthCheckNodeState=${SLURM_HEALTH_CHECK_NODE_STATE}" >> $SLURM_CONF
+      grep -qi '^Prolog' $SLURM_CONF
```

Comment:
If another project installed a prolog script (e.g. Pyxis/Enroot) the NHC kill will never be set in place.
My suggestion would be to install the `kill_nhc.sh` script in `/sched/scripts` and execute such script in a common prolog file (e.g. /sched/scripts/prolog.sh) which is then executed by slurmd.

Reply:
Standardized slurm epilog and prolog amongst different cyclecloud projects, see configure_nhc.sh